app/views/browse/deployment.html

```diff
@@ -1,141 +1,266 @@
+                            </div>
+                          </dd>
+                          <dt>Paused:</dt>
+                          <dd>{{!!deployment.spec.paused}}</dd>
+                        </dl>
+                      </div>
+                      <div class="col-lg-6">
+                        <h3>Configuration</h3>
+                        <dl class="dl-horizontal left">
+                          <dt>Selectors:</dt>
+                          <dd>
+                            <selector selector="deployment.spec.selector"></selector>
+                          </dd>
+                          <dt>Strategy:</dt>
+                          <dd>{{deployment.spec.strategy.type | sentenceCase}}</dd>
+                          <dt ng-if-start="deployment.spec.strategy.rollingUpdate">Max Unavailable:</dt>
+                          <dd>
+                            {{deployment.spec.strategy.rollingUpdate.maxUnavailable}}
+                            <span data-toggle="tooltip"
+                              title="The maximum number of pods that can be unavailable during the update process."
+                              class="pficon pficon-help text-muted small"></span>
+                          </dd>
+                          <dt>Max Surge:</dt>
+                          <dd ng-if-end>
+                            {{deployment.spec.strategy.rollingUpdate.maxSurge}}
+                            <span data-toggle="tooltip"
+                              title="The maximum number of pods that can be created above the desired number of pods."
+                              class="pficon pficon-help text-muted small"></span>
+                          </dd>
+                          <dt>Min Ready:</dt>
+                          <dd>
+                            {{deployment.spec.minReadySeconds || 0}} sec
+                            <span data-toggle="tooltip"
+                              title="The minimum number of seconds for which a newly created pod should be ready without any of its containers crashing for it to be considered available."
+                              class="pficon pficon-help text-muted small"></span>
+                          </dd>
+                        </dl>
+                      </div>
+                    </div>
+                    <div class="row">
+                      <div class="col-lg-6">
+                        <h3>Template</h3>
+                        <pod-template
+                            pod-template="deployment.spec.template"
+                            images-by-docker-reference="imagesByDockerReference"
+                            builds="builds"
+                            detailed="true"
+                            add-health-check-url="{{ ({ group: 'extensions', resource: 'deployments' } | canI : 'update') ? healthCheckURL : '' }}">
```

Comment:
ignore me, i've clearly been looking at this too long already

On Wed, Sep 7, 2016 at 3:08 PM, Sam Padgett <notifications@github.com>
wrote:

> In app/views/browse/deployment.html
> <https://github.com/openshift/origin-web-console/pull/508#discussion_r77884821>
> :
>
> > +                            <span data-toggle="tooltip"
> > +                              title="The minimum number of seconds for which a newly created pod should be ready without any of its containers crashing for it to be considered available."
> > +                              class="pficon pficon-help text-muted small"></span>
> > +                          </dd>
> > +                        </dl>
> > +                      </div>
> > +                    </div>
> > +                    <div class="row">
> > +                      <div class="col-lg-6">
> > +                        <h3>Template</h3>
> > +                        <pod-template
> > +                            pod-template="deployment.spec.template"
> > +                            images-by-docker-reference="imagesByDockerReference"
> > +                            builds="builds"
> > +                            detailed="true"
> > +                            add-health-check-url="{{ ({ group: 'extensions', resource: 'deployments' } | canI : 'update') ? healthCheckURL : '' }}">
>
> If there's no health check URL, the pod template won't link to the action.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-web-console/pull/508/files/1860434f9a525a8ed1f430fe58d2720f9452f79b#r77884821>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABZk7dSffETd-fM3z1R4d1ffaSJe7Cujks5qnwu2gaJpZM4JzBrP>
> .
>